src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -314,26 +370,50 @@ let private createAnnotationTableAtRange (isDark:bool, tryUseLastOutput:bool, ra
 
             // Update annotationTable style
             annotationTable.style <- style
-    
+            annotationTable
+        )
+
+        let _ = table.rows.load(propertyNames = U2.Case2 (ResizeArray[|"count"|]))
+
+        //Skip header because it is newly generated for inMemory table
+        let newColValues =
```

Comment:
We compare values to rows.count, are we sure that rows.count excludes header row?